### PR TITLE
Add stress test program for array and map

### LINKED ISSUE #125: Create program to stress test to determine if data loss is possible
This is meant to run for hours/days using randomized data and operations in random sequences.

### PR DESCRIPTION
**Pull request by fxamacker**
Closes #125

## Description

Add minimal stress test program to detect data loss.

This is not a fuzz test.

- [x] stress test array with `Append`, `Insert`, `Set`, and `Remove` ops
- [x] stress test map with `Set` and `Remove` ops

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/atree/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: cmd/stress/utils.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+func randStr(n int) string {
+	r := make([]rune, n)
+	for i := range r {
+		r[i] = letters[rand.Intn(len(letters))]
+	}
+	return string(r)
+}
+
+// TODO: add nested array and nested map
```
Comment (turbolent) [inline cmd/stress/utils.go:39]: That would be a really good idea and good next task 👍

**Review by turbolent — COMMENTED** [review]
Nice, tn

**Review by turbolent — APPROVED** [review]
Looks good! Would be great to combine the two modes and have arrays and maps be children of each other at arbitrary deeply nesting

**Comment by fxamacker** [comment]
> Looks good! Would be great to combine the two modes and have arrays and maps be children of each other at arbitrary deeply nesting

I opened issue #157 for this.  I think it may require deep remove (currently implemented in Cadence).

**Pull request merged**